networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -0,0 +1,72 @@
+// ExternalDNS Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="external-dns-operator-release-notes"]
+= ExternalDNS Operator release notes
+:context: external-dns-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The ExternalDNS Operator deploys and manages `ExternalDNS` to provide the name resolution for services and routes from the external DNS provider to OpenShift Container Platform.
```

Comment:
"The" is unnecessary before "name resolution."

Reply:
check